cmd/controlplane/management_controller.go

```diff
@@ -54,6 +60,9 @@ func newManagementControllerCommand() *cobra.Command {
 
 func (o *managementControllerOptions) complete() {
 	o.KubeConfig = os.GetEnv("KUBECONFIG", "")
+	o.KargoNamespace = os.GetEnv("KARGO_NAMESPACE", "kargo")
+	o.ManageControllerRoleBindings = types.MustParseBool(os.GetEnv("MANAGE_CONTROLLER_ROLE_BINDINGS", "false"))
```

Comment:
```suggestion
	o.ManageControllerRoleBindings = types.MustParseBool(os.GetEnv("MANAGE_CONTROLLER_ROLE_BINDINGS", "true"))
```

This should be the default behavior -- enabled unless not _explicitly_ disabled.